### PR TITLE
Include 2014 fluxes for IC (solmax) and SK (solmin and solmax) for integral preserving fluxes

### PR DESCRIPTION
**Pull request by pabloferm**
Enable below GeV computation for SK fluxes.

### CODE REVIEW

File: src/library/IPLEFlux.cpp [hunk 1]
```diff
@@ -81,7 +86,8 @@ namespace nuflux{
   }
 
   double IntegralPreservingFlux::getMinEnergy() const{
-    return 1;
+    if (SetSubGeV==true) return 1e-1;
```
Comment (kjmeagher) [inline src/library/IPLEFlux.cpp:89]: Please use curly braces here

**Review by kjmeagher — CHANGES_REQUESTED** [review]

File: src/library/IPLEFlux.cpp [hunk 2]
```diff
@@ -25,6 +25,11 @@ namespace nuflux{
   IntegralPreservingFlux::~IntegralPreservingFlux(){}
   
   void IntegralPreservingFlux::loadTables(const std::string& fluxName, ParticleType type){
+
+    SetSubGeV = fluxName.find("sk")  != std::string::npos; // Extend to SubGeV for fluxes at SK
+//    SetSubGeV = fluxName.find("sno") != std::string::npos; // Extend to SubGeV for fluxes at SNO
```
Comment (kjmeagher) [inline src/library/IPLEFlux.cpp:30]: Don't add dead code

File: src/library/IPLEFlux.cpp (hunk 1 above)
Comment (pabloferm) [reply src/library/IPLEFlux.cpp]: Done

File: src/library/IPLEFlux.cpp (hunk 2 above)
Comment (pabloferm) [reply src/library/IPLEFlux.cpp]: Removed

**Pull request merged**